src/main/java/github/scarsz/discordsrv/objects/managers/GroupSynchronizationManager.java

```diff
@@ -178,6 +178,12 @@ public void resync(OfflinePlayer player, SyncDirection direction, boolean addLin
 
         Map<Guild, Map<String, Set<Role>>> roleChanges = new HashMap<>();
 
+        // Check if Minecraft or Discord is strictly authoritative.
+        boolean oneWaySynchronisation = DiscordSRV.config().getBoolean("GroupRoleSynchronizationOneWay");
+        boolean minecraftIsStrictlyAuthoritative = oneWaySynchronisation && DiscordSRV.config().getBoolean("GroupRoleSynchronizationMinecraftIsAuthoritative");
+        boolean discordIsStrictlyAuthoritative = oneWaySynchronisation && !DiscordSRV.config().getBoolean("GroupRoleSynchronizationMinecraftIsAuthoritative");
+        synchronizationSummary.add("Synchronisation is one way (" + (minecraftIsStrictlyAuthoritative ? "MineCraft -> Discord" : "Discord -> MineCraft") + ")");
```

Comment:
Use proper capitalization of `Minecraft`

Reply:
Done!